docs/content/node-operation/node-bootstrap.mdx

```diff
@@ -102,24 +103,48 @@ $./boot-tools/bootstrap key --address "198.123.42.10:3569" --role consensus  -o
 <nil> INF wrote file bootstrap/public-root-information/node-id
-<nil> INF wrote file bootstrap/private-root-information/private-node-info_39fa54984b8eaa463e129919464f61c8cec3a4389478df79c44eb9bfbf30799a/node-info.priv.json
-<nil> INF wrote file bootstrap/public-root-information/node-info.pub.39fa54984b8eaa463e129919464f61c8cec3a4389478df79c44eb9bfbf30799a.json
+<nil> INF wrote file bootstrap/private-root-information/private-node-info_ab6e0b15837de7e5261777cb65665b318cf3f94492dde27c1ea13830e989bbf9/node-info.priv.json
+<nil> INF wrote file bootstrap/public-root-information/node-info.pub.ab6e0b15837de7e5261777cb65665b318cf3f94492dde27c1ea13830e989bbf9.json
+<nil> DBG will generate machine account key
+<nil> INF generated machine account key
+<nil> DBG assembling machine account information address=198.123.42.10:3569
+<nil> DBG encoded public machine account key machineAccountPubKey=1b9c00e6f0930792c5738d3397169f8a592416f334cf11e84e6327b98691f2b72158b40886a4c3663696f96cd15bfb5a08730e529f62a00c78e2405013a6016d
+<nil> INF wrote file bootstrap/private-root-information/private-node-info_ab6e0b15837de7e5261777cb65665b318cf3f94492dde27c1ea13830e989bbf9/node-machine-account-key.priv.json
 
 $tree ./bootstrap/
-./bootstrap/
+./bootstrap
 ├── private-root-information
-│   └── private-node-info_39fa54984b8eaa463e129919464f61c8cec3a4389478df79c44eb9bfbf30799a
-│       └── node-info.priv.json
+│   └── private-node-info_ab6e0b15837de7e5261777cb65665b318cf3f94492dde27c1ea13830e989bbf9
+│       ├── node-info.priv.json
+│       └── node-machine-account-key.priv.json
 └── public-root-information
     ├── node-id
-    └── node-info.pub.39fa54984b8eaa463e129919464f61c8cec3a4389478df79c44eb9bfbf30799a.json
+    └── node-info.pub.ab6e0b15837de7e5261777cb65665b318cf3f94492dde27c1ea13830e989bbf9.json
+
+3 directories, 4 files
+```
+
+<Callout type="warning">
+For `consensus` and `collection` node types a key known as the Machine account will be generated. For all other node types this will not be needed.
+</Callout>
 
-3 directories, 3 files
+#### Generate Machine Account Address
 
+Once you have generated all your keys, you will need to now submit a transaction on Flow Port to create a Machine Account for the Machine Account public key. In the output of 
+the `boostrap key` command from the previous step you will see your Machine Account public key.
+
+```shell:title=Example
+$./boot-tools/bootstrap key --address "198.123.42.10:3569" --role consensus  -o ./bootstrap
+...
+<nil> DBG encoded public machine account key machineAccountPubKey=1b9c00e6f0930792c5738d3397169f8a592416f334cf11e84e6327b98691f2b72158b40886a4c3663696f96cd15bfb5a08730e529f62a00c78e2405013a6016d
+<nil> INF wrote file bootstrap/private-root-information/private-node-info_ab6e0b15837de7e5261777cb65665b318cf3f94492dde27c1ea13830e989bbf9/node-machine-account-key.priv.json
 ```
 
+You can follow the guide [here](/flow-port/staking-guide) to submit a transaction on [Flow Port]().
```

Comment:
The `Flow Port` link is empty -- guessing this should point to https://port.onflow.org/

Reply:
```suggestion
You can follow the guide [here](/flow-port/staking-guide) to submit a transaction on [Flow Port](https://port.onflow.org/).
```